pkg/build/generated/internalclientset/typed/build/internalversion/scheme_expansion.go

```diff
@@ -0,0 +1,10 @@
+package internalversion
+
+import (
+	buildv1 "github.com/openshift/origin/pkg/build/apis/build/v1"
+	build "github.com/openshift/origin/pkg/build/generated/internalclientset/scheme"
+)
+
+func init() {
+	buildv1.RegisterConversions(build.Scheme)
```

Comment:
@liggitt without this you get https://gist.github.com/mfojtik/9c4b8bc6aba9055309dc4935477951ee

I double checked, you added the RegisterConversions to buildv1.SchemeBuilder and LegacySchemeBuilder so I have no idea why it is not seeing them.